ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -638,8 +640,41 @@ static void HandleKernelRequest(void* messageMemory, uint32_t messageSize)
     
     Message request = ParseMessageMemory(messageMemory, messageSize);
     
-    // At the moment, we expect all messages to include a path
-    assert(request.path != nullptr);
+    char pathBuffer[PrjFSMaxPath];
+    if (request.path == nullptr)
+    {
+        fsid_t fsid = request.messageHeader->fsidInode.fsid;
+        ssize_t pathSize = fsgetpath(pathBuffer, sizeof(pathBuffer), &fsid, request.messageHeader->fsidInode.inode);
+        if (pathSize < 0)
+        {
+            cout
```

Comment:
This looks like somewhere that #395 would be helpful.

Would you mind adding a comment here with a TODO?  Something like:

```
// TODO(Mac): Add this message to PrjFSLib logging once available (#395)
```

/cc: @jrbriggs @jeschu1 @nickgra as FYI